### ISSUE TITLE
Proposal guidelines: advice to mind target audience

### ISSUE THREAD

**Issue by xaur** [decred/dcrdocs#876]
I noticed that some proposal authors don't properly consider target audience (stakeholders). Two examples that come to mind are Ditto and Wachsman proposals (used too much cryptic marketing jargon without explaining it), and the DEX proposal (used the ((double parens)) notation for concatenation).

When appealing to Decred stakeholders, there's no "Marketing boss" or "Dev boss" you can talk to who knows the domain. You talk to a crowd of random people, many may be illiterate in your domain. Try to dedicate at least a small part of the proposal for them explaining why it is good for them ELI5 style.

This is a bit related to internationalization issue: do not use over-complicated English that some stakeholders may not understand.

**Comment by s-ben** [comment]
I definitely think we should add this as a tip on the [Proposal Guidelines](https://docs.decred.org/governance/politeia/proposal-guidelines/) page. 

We can also review the [Example Proposals](https://docs.decred.org/governance/politeia/example-proposals/) and make sure they're up to this standard. 

> This is a bit related to internationalization issue: do not use over-complicated English that some stakeholders may not understand.

Have totally started thinking about this one as I write too...have held my slang back a few times.
